packages/scaffold-ui-new/src/utils/UiHelperUtil.ts

```diff
@@ -0,0 +1,35 @@
+import { ConstantsUtil } from './ConstantsUtil.js'
```

Comment:
We are doing JS based animation for screens, this is generally a pain points when it comes to SSR. When you open the page on mobile, the server uses the default value as breakpoint while on client it uses mobile breakpoint. 

We should definitely make sure it's not going to run into that stage. @magiziz